### DISCUSSION TITLE
Trouble with upgrade to 8.0

### DISCUSSION THREAD

**Discussion by Fullyfly** [iNavFlight/inav#10525] [Support] [answered]
Hi All,
So installed Inav 6 a while ago and never had the chance to finish the project. Up and flying now and thought I should upgrade to latest version. I did a Diff All in the CLI and saved that. Upgraded to 8 and copied it back.
There is a note saying "Profile 1" needs changing to "control_profile 1" How do I edit the CLI or update the profile? I tried editing the text file but just get a greyed out screen when I try to open it. 
Also has some OSD settings that need updating. I assume the same process needed. At the moment when I got to the OSD page it says waiting for data then just a blank OSD screen. I assume because I have to make the updates but not sure how to change.
Any advice appreciated.
Cheers

**Comment by mmosca** [comment, marked as answer]
Any text editor, like notepad should allow to edit the diff.

But if you are moving from 6, also check the release and update notes for INAV 7.

**Reply by Fullyfly** [reply]
Hi Mmosca.
Thanks for the tips. I went and checked the update from 6 to 7, Only a note about changing font in the OSD tap. At the moment that is all blank Im thinking because of the 7 to 8 change I still haven't managed to make. 
For the control_Profile 1 change I have edited in notepad and also notepad ++ and copied into the CLI and hit enter. When it runs through it has a error at this line and put a # in front of it. When I check what was pasted in the command line it doesn't have the #. I have also tried the "Load from file" and execute option and comes up with the same error. 
See attached pic. You can see the error and see in the command line where I'm sure it looks correct. 
Im sure im missing something obvious. Bit frustrating I did a lot of RC stuff with Heli, Planes, Quads etc for several years. Nothing over the last 5 years as got stuck out of the country I was living over covid. Now I have all my RC stuff back and getting back into it. Everything has change soooooo much and I have forgotten sooooo much. 
Appreciate the help.
Cheers
![CLI](https://github.com/user-attachments/assets/1e376a22-0d27-4129-b28c-8d689af326cd)

**Comment by Fullyfly** [comment]
ok might have found the problem. I have downloaded the 8.0 configurator and when I flash there is no 8.0 version for my speedybee F405v3
Looks like I should go back to version 7 of the configurator
![speedy FW](https://github.com/user-attachments/assets/3fede1a4-6bda-467f-94a6-e4fa43aff0fa)
.

**Reply by sensei-hacker** [reply]
If you want to use the pre-release 8.0RC1, you must check the box marked "show unstable releases".
